routes/api/v1/twilio.js

```diff
@@ -156,10 +168,11 @@ function getLanguageString(language, title, data) {
 function matchYes(req, language) {
   var input = req.body.Body.toUpperCase().trim();
   var yes = getLanguageYes(language, true);
+  debug('matchYes, input:', input);
   if (_.isArray(yes)) {
     return _.contains(yes, input);
   } else {
-    return input === getLanguageYes(language);
+    return input === yes;
```

Comment:
nice, but i wonder if the configuration definitions should just have `yes` be an array regardless of if there's only one `yes` (i.e. `yes: ["YES"]`) or multiple.  perhaps a future change